pages/common/lldb.md

```diff
@@ -0,0 +1,15 @@
+# lldb
+
+> The LLDB Debugger.
```

Comment:
You can write the expanded version in brackets - LLDB(Low Level Debugger). That is a norm we follow in various pages.